### ISSUE TITLE
ROMBASE, MEMBASE, MEMSIZE changes do not affect dependencies

### ISSUE THREAD

**Issue by swetland** [travisg/lk#49]
In particular the generated linker scripts will not be updated if these #defines change, which results in bogus binaries.

**Comment by M1cha** [comment]
I can confirm this and afaik travis already knows about this.

**Comment by swetland** [comment]
Yup, was chatting with him about it this afternoon.  Figured it wouldn't hurt to start tracking known issues.

**Comment by travisg** [comment]
I think this is fixed now since it always regens the linker script and updates it if any of the input vars changed.
